README.md

```diff
@@ -2,197 +2,68 @@
-| **v0.1.1-alpha** | v0.11.0-beta | v0.8.1-beta | v0.2.0-alpha |
-| **v0.1.0-alpha** | v0.10.3-beta | v0.6.5-beta | v0.2.0-alpha |
-
-## Usage
+![screenshot](./app/src/assets/images/screenshot.png)
 
-Read the [Walkthrough](doc/WALKTHROUGH.md) document to learn more about how to use
-Lightning Terminal.
+## Features
+- Visualize your channels and balances
+- Perform submarine swaps via the [Lightning Loop](https://lightning.engineering/loop) service
+- Classify channels according to your node's operating mode
+- Run a single binary that integrates both [`loopd`](https://github.com/lightninglabs/loop) and [`faraday`](https://github.com/lightninglabs/faraday) daemons all in one
 
 ## Installation
+Download the latest binaries from the [releases](https://github.com/lightninglabs/lightning-terminal/releases) page.
 
-There are two options for installing LiT: download the published binaries for your
-platform, or compile from source code.
-
-#### Download Binaries
-
-LiT binaries for many platforms are made available on the GitHub
-[Releases](https://github.com/lightninglabs/lightning-terminal/releases) page in this
-repo. There you can download the latest version and extract the archive into a directory
-on your computer.
-
-#### Compile from Source Code
-
-To compile from source code, you'll need to have some prerequisite developer tooling
-installed on your machine.
-
-| Dependency                                          | Description                                                                                                                                                                          |
-| --------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| [golang](https://golang.org/doc/install)            | LiT's backend web server is written in Go. The minimum version supported is Go v1.13.                                                                                                |
-| [protoc](https://grpc.io/docs/protoc-installation/) | Required to compile LND & Loop gRPC proto files at build time                                                                                                                        |
-| [nodejs](https://nodejs.org/en/download/)           | LiT's frontend is written in TypeScript and built on top of the React JS web framework. To bundle the assets into Javascript & CSS compatible with web browsers, NodeJS is required. |
-| [yarn](https://classic.yarnpkg.com/en/docs/install) | a popular package manager for NodeJS application dependencies                                                                                                                        |
-
-Once you have the necessary prerequisites, LiT can be compiled by running the following
-commands:
+## Execution
+Run Lightning Terminal with a local LND instance:
```

Comment:
In my parts of the documentation I always use the `lnd` notation (lower case and with back ticks to mark as inline code). Not sure what the "preferred" notation is but we should probably try to use the same notation everywhere.

Reply:
Fixed